src/app/app.routes.ts

```diff
@@ -0,0 +1,18 @@
+import { Routes } from '@angular/router';
+
+import { InvoiceListComponent } from '../app/pages/invoice-list/invoice-list.component';
+import { AuthGuard } from './core/auth.guard';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
+import { EditVoiceComponent } from './shared/edit-voice/edit-voice.component';
+import { UserComponent } from './user/user.component';
+import { UserResolver } from './user/user.resolver';
+
+export const rootRouterConfig: Routes = [
+  { path: '', redirectTo: 'login', pathMatch: 'full' },
+  { path: 'login', component: LoginComponent, canActivate: [AuthGuard] },
```

Comment:
why auth guard is used in login and register component? Use auth guard on landing component so that unauthorized user will be redirected to login page